src/SQLStore/QueryDependency/DependencyLinksValidator.php

```diff
@@ -119,7 +119,7 @@ public function hasArchaicDependencies( DIWikiPage $subject ) {
 		$id_table = $connection->tableName( SQLStore::ID_TABLE );
 
 		$rows = $connection->select(
```

Comment:
I cannot point to the right lines 136-137 since they are not edited, but in the last parameter (join_conditions) it would be better to use only as key the table name or the alias (see [MW doc](https://github.com/wikimedia/mediawiki/blob/a11f20989b829f4ede6c731e6126eea3defdbba6/includes/libs/rdbms/database/IReadableDatabase.php#L415-L428)). Given the same table is used multiple times here, I guess the alias is better here to distinguish the two same tables.

Consequently, the variable `$id_table` just above should be deleted.

Reply:
Oh, so just using `"p" => [ "INNER JOIN" ... ]` on its own in the join condition is fine if I understand this right? I was wondering about whether that can be cleaned up as well, will try that tomorrow.

Reply:
implemented as you suggested.